code/game/objects/effects/map_effects/mapmanip.dm

```diff
@@ -20,7 +20,15 @@
 	pixel_x = -32
 	pixel_y = -32
 
+/obj/effect/map_effect/marker_helper
+	name = "marker helper"
+	layer = POINT_LAYER
+
+/obj/effect/map_effect/marker_helper/Initialize(mapload)
+	. = ..()
+	qdel(src)
```

Comment:
This should do `return INITIALIZE_HINT_QDEL`

Reply:
done, PTAL